packages/okta-vue/README.md

```diff
@@ -1,5 +1,8 @@
 # Okta Vue SDK
 
+[![npm version](https://img.shields.io/npm/v/@okta/okta-vue.svg?style=flat-square)](https://www.npmjs.com/package/@okta/okta-vue)
+[![build status](https://img.shields.io/travis/okta/okta-oidc-js/master.svg?style=flat-square)](https://travis-ci.org/okta/okta-oidc-js)
```

Comment:
Since all these tests are connected, would it make more sense to bring this out in the primary `README`?

Reply:
I see your request and I raise you a markdown table!  See what I just pushed and let me know what you think!